submissions/end-2021/end-2021.md

```diff
@@ -0,0 +1,84 @@
+
+> The first time I played I had had this weird idea to loop back my resources back to the start to not waste them. It did not work well. - stringweasel, Alt-F4 Editor
+
+> I tried out the demo and ended up playing for an entire day. I spent ages mining out the entire tiny demo map and working out how to cross over belts without undergrounds using inserters. That night, I slept terribly, and dreamed of nothing but factories. I bought the full game first thing the next morning. - Bentham, [Content Creator](https://alt-f4.blog/ALTF4-36/)
+
+> New updates with extra content are what I miss the most, rocket silo with infinite science, nuclear update, artillery and many small changes was awesome during beta. - Trupen, [Content Creator](https://www.youtube.com/c/Trupen)
+
+> One of my most memorable and funny moments is from one of the Earth Map pkaythroughs. Colonel Will was "gardening" with the Orbital Ion Cannon Mod and decided it to "trim down" some trees near me. When he warned me with about 3 seconds before impact, I panicked ran straight into the blast! - Xterminator, [Content Creator](https://www.youtube.com/channel/UC5StrkKVnU2xkjV0mVJ9yTw)
+
+> idk, first time I did oil in 0.11 was pretty magical and I did the thing with logistic bots where they craft themselves. - Klonan, Wube Developer
+
+![We end of by showing you Klonan's first base!](media/klonan-base.png)
+
+> There lies a hidden gem within Factorio we have not yet fully explored.  I will never forget how much I enjoyed the multiplayer Production Battle PvP scenario that was played years ago.  Points are awarded for anything you make and removed for anything you consume.  My all time favorite moment of Factorio was that moment that our team's score went from positive 60k to negative 60k points per minute and everyone who was watching said, "What the heck is going on?!" - Rain, [Content Creator and Speedrunner](https://www.twitch.tv/rain9441)
+
+> "The game design is so complex, they must have some crazy spreadsheets and tools to balance all this out" was also something I thought before working on the game. Then I started working and realized it was all balanced and designed by just playing the game. - Twinsen, Wube Developer
+
+> I remember that I almost didn't pick up Factorio because of the basic graphics, but the trailer was amazing at showing the ever growing scale that I've come to love about the game. What I really enjoyed in the pre-release era was the constant evolution of the game; new core designs were invented regularly (like Main Bus and City Blocks), weekly blogs and regular new features added (Nuclear and Infinite Science), and big overhaul mods reaching maturity (Bobs, Angels and AAI). I hope the communication from Wube starts soon for the expansion, so we can reignite the magic of being part of the evolution of game we love" - Nilaus, [Content Creator](https://www.youtube.com/c/Nilaus)
```

Comment:
```suggestion
> I remember that I almost didn't pick up Factorio because of the basic graphics, but the trailer was amazing at showing the ever-growing scale that I've come to love about the game. What I really enjoyed in the pre-release era was the constant evolution of the game; new core designs were invented regularly (like the Main Bus and City Blocks), there were weekly blogs, new features were added regularly (Nuclear and Infinite Science), and big overhaul mods were reaching maturity (Bobs, Angels and AAI). I hope the communication from Wube starts soon for the expansion, so we can reignite the magic of being part of the evolution of the game we love." - Nilaus, [Content Creator](https://www.youtube.com/c/Nilaus)
```